enhancements/node-tuning/machine-config-support-in-nto.md

```diff
@@ -0,0 +1,111 @@
+---
+title: machine-config-support-in-nto
+authors:
+  - "@jmencak"
+reviewers:
+  - "@MarSik"
+  - "@sjug"
+  - "@slintes"
+  - "@yanirq"
+  - "@zvonkok"
+approvers:
+  - "@MarSik"
+  - "@sjug"
+  - "@slintes"
+  - "@yanirq"
+  - "@zvonkok"
+
+creation-date: 2020-03-26
+last-updated: 2020-03-26
+status: **provisional**|implementable|implemented|deferred|rejected|withdrawn|replaced
+---
+
+# Machine Config Support in NTO
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes adding the ability for admins to target MachineConfigPools
+and create MachineConfigs through the Node Tuning Operator.  With this enhancement we
+can take another step towards full support for tuned profiles that need `[bootloader]`
+support on RHCOS.
+
+## Motivation
+
+In support of Telco 5G customers, partial tuned real-time profile support
+already landed OpenShift.  However, a lot of node-management tasks still need to be
+performed by the OCP administrator or an addon operator, so the current profile
+delivery is not fully automated and performed from a centralized location.
+
+Specialized profiles, such as cpu-partitioning or realtime exist on RHEL which
+deliver specialized tuning.  These profiles perform calculations based on a
+variable input and, among other things, set proper kernel boot parameters
+to achieve requested tuning.
+
+These tuned profiles are written with RHEL rather than RHCOS in mind, therefore
+host modifications such as kernel boot parameters becomes problematic on RHCOS.
+Support for MachineConfigPools/MachineConfigs in the Node Tuning Operator can 
+overcome these issues on RHCOS.
+
+### Goals
+
+- [ ] The ability to target MachineConfigPools from the Tuned API and create
```

Comment:
Maybe add a hyper-link for Tuned API  so we can understand its an API coming from NTO and not tuned iteself ?

Reply:
Thank you, adjusted.